Enable custom roles and spaces in serverless projects

jeramysoucy opened the pull request:
Closes #194933
Closes #192282

## Summary

This PR updates the serverless project yml files to

- enable custom roles for Elasticsearch and Security projects
- enable multiple spaces (max 100) for all serverless project types

### Tests
Additionally, this PR adjust the serverless test suites. Originally, testing of roles and spaces endpoints was achieved from the feature flag test config. Now that these features are enabled by default, the tests have been migrated to the standard serverless test configs.

Affected tests:
- x-pack/test_serverless/api_integration/test_suites/common/management/spaces.ts
- x-pack/test_serverless/api_integration/test_suites/common/platform_security/authorization.ts
- x-pack/test_serverless/functional/test_suites/common/platform_security/navigation/management_nav_cards.ts
- x-pack/test_serverless/functional/test_suites/common/platform_security/roles.ts
- x-pack/test_serverless/functional/test_suites/common/spaces/spaces_management.ts
- x-pack/test_serverless/functional/test_suites/common/spaces/spaces_selection.ts
- Feature flag configs/indices
- Project specific configs/indices
- Base serverless config

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/240673)
* Commit: 998c76ffda8f2eb0d42511898c9df5b1f1ccea28

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/240673#01927148-24c8-4269-b8e5-3c5b08d23578)

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/240673","number":240673,"commit":"998c76ffda8f2eb0d42511898c9df5b1f1ccea28"}],"number":240673}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/240827)
* Commit: b7286c2829334cb34f6ab55f5b403f24f966f22c

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/240827#01927237-1344-4a80-b0fc-8db334cc69fd)

### History
* :broken_heart: [Build #240673](https://buildkite.com/elastic/kibana-pull-request/builds/240673) failed 998c76ffda8f2eb0d42511898c9df5b1f1ccea28

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/240827","number":240827,"commit":"b7286c2829334cb34f6ab55f5b403f24f966f22c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/240673","number":240673,"commit":"998c76ffda8f2eb0d42511898c9df5b1f1ccea28"}],"number":240827}
buildkite-pr-comment-->

jeramysoucy commented on x-pack/test_serverless/api_integration/test_suites/observability/config.feature_flags.ts:
```diff
@@ -22,13 +22,12 @@ export default createTestConfig({
     '--xpack.infra.enabled=true',
-    '--xpack.security.roleManagementEnabled=true', // enables custom roles
-    `--xpack.spaces.maxSpaces=100`, // enables spaces UI capabilities
+    '--xpack.security.roleManagementEnabled=true', // needed to check composite feautures in /observability/platform_security/authorization.ts
```
@azasypkin Should we expose the `api/security/privileges` endpoint in serverless even if role management is off? Currently, we do not.

azasypkin replied:
Hmm, I’d say no, we shouldn’t. But would you mind giving me a bit of context? Do I understand correctly that we’ll still keep role/privilege-related tests for O11y, but only via the “feature_flags” config (unlike other project types for which we can finally move to base/normal configs)?

azasypkin replied:
> but this endpoint is only available when we enable role management so it needs to remain in the feature flag config until either oblt enabled custom roles

Ah, got it, thanks, it makes sense to me 👍

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/241063)
* Commit: 7236e7acb8e5dbe8c7e857bc543e41c3511a31e1

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/241063#019275e7-3ebf-41c5-b1f9-3bf7f92cab1f)

### History
* :broken_heart: [Build #240827](https://buildkite.com/elastic/kibana-pull-request/builds/240827) failed b7286c2829334cb34f6ab55f5b403f24f966f22c
* :broken_heart: [Build #240673](https://buildkite.com/elastic/kibana-pull-request/builds/240673) failed 998c76ffda8f2eb0d42511898c9df5b1f1ccea28

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241063","number":241063,"commit":"7236e7acb8e5dbe8c7e857bc543e41c3511a31e1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/240827","number":240827,"commit":"b7286c2829334cb34f6ab55f5b403f24f966f22c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/240673","number":240673,"commit":"998c76ffda8f2eb0d42511898c9df5b1f1ccea28"}],"number":241063}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/241272)
* Commit: 1994ad6c7c2df9fae2fe8a53aa9539adaf8faf8e

### Failed CI Steps
* [FTR Configs #57](https://buildkite.com/elastic/kibana-pull-request/builds/241272#01927747-0cc0-4618-b67e-9a61577d5e31)
* [FTR Configs #87](https://buildkite.com/elastic/kibana-pull-request/builds/241272#01927747-0d08-4181-b459-51bf15c3548c)
* [FTR Configs #87](https://buildkite.com/elastic/kibana-pull-request/builds/241272#0192776e-6404-43bf-81d1-db3a4f02f2d9)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/241272#01927747-0cc0-4618-b67e-9a61577d5e31) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/241272/jobs/01927747-0cc0-4618-b67e-9a61577d5e31/artifacts/01927770-a770-4fcd-af0f-69cf4925c502) FTR Configs #<span></span>57 / serverless search UI - feature flags Search index detail page index details page overview should have connection details

### History
* :broken_heart: [Build #241063](https://buildkite.com/elastic/kibana-pull-request/builds/241063) failed 7236e7acb8e5dbe8c7e857bc543e41c3511a31e1
* :broken_heart: [Build #240827](https://buildkite.com/elastic/kibana-pull-request/builds/240827) failed b7286c2829334cb34f6ab55f5b403f24f966f22c
* :broken_heart: [Build #240673](https://buildkite.com/elastic/kibana-pull-request/builds/240673) failed 998c76ffda8f2eb0d42511898c9df5b1f1ccea28

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241272","number":241272,"commit":"1994ad6c7c2df9fae2fe8a53aa9539adaf8faf8e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241063","number":241063,"commit":"7236e7acb8e5dbe8c7e857bc543e41c3511a31e1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/240827","number":240827,"commit":"b7286c2829334cb34f6ab55f5b403f24f966f22c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/240673","number":240673,"commit":"998c76ffda8f2eb0d42511898c9df5b1f1ccea28"}],"number":241272}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/241426)
* Commit: 2bb076664ad69fd973837fb56ec3d3c7c882088a

### Failed CI Steps
* [FTR Configs #62](https://buildkite.com/elastic/kibana-pull-request/builds/241426#01927a6d-e692-4f75-b7e8-ff7f0f91f48f)
* [FTR Configs #62](https://buildkite.com/elastic/kibana-pull-request/builds/241426#01927a94-a203-46a4-b42e-af0cc901a2c1)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/241426#01927a6d-e692-4f75-b7e8-ff7f0f91f48f) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/241426/jobs/01927a6d-e692-4f75-b7e8-ff7f0f91f48f/artifacts/01927a94-84f2-48cd-b516-060291ba0841) FTR Configs #<span></span>62 / serverless search UI - feature flags Search index detail page index details page overview should have connection details
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/241426#01927a94-a203-46a4-b42e-af0cc901a2c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/241426/jobs/01927a94-a203-46a4-b42e-af0cc901a2c1/artifacts/01927a9f-d52f-4edd-b7ca-04639f0e1856) FTR Configs #<span></span>62 / serverless search UI - feature flags Search index detail page index details page overview should have connection details

### History
* :broken_heart: [Build #241272](https://buildkite.com/elastic/kibana-pull-request/builds/241272) failed 1994ad6c7c2df9fae2fe8a53aa9539adaf8faf8e
* :broken_heart: [Build #241063](https://buildkite.com/elastic/kibana-pull-request/builds/241063) failed 7236e7acb8e5dbe8c7e857bc543e41c3511a31e1
* :broken_heart: [Build #240827](https://buildkite.com/elastic/kibana-pull-request/builds/240827) failed b7286c2829334cb34f6ab55f5b403f24f966f22c
* :broken_heart: [Build #240673](https://buildkite.com/elastic/kibana-pull-request/builds/240673) failed 998c76ffda8f2eb0d42511898c9df5b1f1ccea28

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241426","number":241426,"commit":"2bb076664ad69fd973837fb56ec3d3c7c882088a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241272","number":241272,"commit":"1994ad6c7c2df9fae2fe8a53aa9539adaf8faf8e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241063","number":241063,"commit":"7236e7acb8e5dbe8c7e857bc543e41c3511a31e1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/240827","number":240827,"commit":"b7286c2829334cb34f6ab55f5b403f24f966f22c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/240673","number":240673,"commit":"998c76ffda8f2eb0d42511898c9df5b1f1ccea28"}],"number":241426}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/241517)
* Commit: 5cee6b25d96f5bc2f61c61bb40de56b536d7d45c

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :broken_heart: [Build #241426](https://buildkite.com/elastic/kibana-pull-request/builds/241426) failed 2bb076664ad69fd973837fb56ec3d3c7c882088a
* :broken_heart: [Build #241272](https://buildkite.com/elastic/kibana-pull-request/builds/241272) failed 1994ad6c7c2df9fae2fe8a53aa9539adaf8faf8e
* :broken_heart: [Build #241063](https://buildkite.com/elastic/kibana-pull-request/builds/241063) failed 7236e7acb8e5dbe8c7e857bc543e41c3511a31e1
* :broken_heart: [Build #240827](https://buildkite.com/elastic/kibana-pull-request/builds/240827) failed b7286c2829334cb34f6ab55f5b403f24f966f22c
* :broken_heart: [Build #240673](https://buildkite.com/elastic/kibana-pull-request/builds/240673) failed 998c76ffda8f2eb0d42511898c9df5b1f1ccea28

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241517","number":241517,"commit":"5cee6b25d96f5bc2f61c61bb40de56b536d7d45c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241426","number":241426,"commit":"2bb076664ad69fd973837fb56ec3d3c7c882088a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241272","number":241272,"commit":"1994ad6c7c2df9fae2fe8a53aa9539adaf8faf8e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241063","number":241063,"commit":"7236e7acb8e5dbe8c7e857bc543e41c3511a31e1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/240827","number":240827,"commit":"b7286c2829334cb34f6ab55f5b403f24f966f22c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/240673","number":240673,"commit":"998c76ffda8f2eb0d42511898c9df5b1f1ccea28"}],"number":241517}
buildkite-pr-comment-->

elasticmachine commented:
Pinging @elastic/kibana-security (Team:Security)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/241972)
* Commit: 3dc49a470916ba0958a28410606a5ea59cfa8fa9

### Failed CI Steps
* [Build Kibana Distribution and Plugins](https://buildkite.com/elastic/kibana-pull-request/builds/241972#01928a1d-a4ac-44df-aa06-a19bdc8aad19)

### History
* :broken_heart: [Build #241949](https://buildkite.com/elastic/kibana-pull-request/builds/241949) failed e72cf087d6b1a2f4a0ad788b0d0201307a5ecdb8
* :green_heart: [Build #241517](https://buildkite.com/elastic/kibana-pull-request/builds/241517) succeeded 5cee6b25d96f5bc2f61c61bb40de56b536d7d45c
* :broken_heart: [Build #241426](https://buildkite.com/elastic/kibana-pull-request/builds/241426) failed 2bb076664ad69fd973837fb56ec3d3c7c882088a
* :broken_heart: [Build #241272](https://buildkite.com/elastic/kibana-pull-request/builds/241272) failed 1994ad6c7c2df9fae2fe8a53aa9539adaf8faf8e
* :broken_heart: [Build #241063](https://buildkite.com/elastic/kibana-pull-request/builds/241063) failed 7236e7acb8e5dbe8c7e857bc543e41c3511a31e1

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241972","number":241972,"commit":"3dc49a470916ba0958a28410606a5ea59cfa8fa9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241949","number":241949,"commit":"e72cf087d6b1a2f4a0ad788b0d0201307a5ecdb8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241517","number":241517,"commit":"5cee6b25d96f5bc2f61c61bb40de56b536d7d45c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241426","number":241426,"commit":"2bb076664ad69fd973837fb56ec3d3c7c882088a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241272","number":241272,"commit":"1994ad6c7c2df9fae2fe8a53aa9539adaf8faf8e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241063","number":241063,"commit":"7236e7acb8e5dbe8c7e857bc543e41c3511a31e1"}],"number":241972}
buildkite-pr-comment-->

jloleysens reviewed (approved):
Nice work @jeramysoucy !

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/242003)
* Commit: 3d77eaab53b3b3d7bf2a019c5756a9cde09653fb

### Failed CI Steps
* [FTR Configs #48](https://buildkite.com/elastic/kibana-pull-request/builds/242003#01928a62-597c-4989-8530-a2d8b36dab6b)
* [FTR Configs #56](https://buildkite.com/elastic/kibana-pull-request/builds/242003#01928a62-598c-43e7-be48-6993307ceecf)
* [FTR Configs #56](https://buildkite.com/elastic/kibana-pull-request/builds/242003#01928a87-7d2b-41e4-b3b2-fadcd886a8af)
* [FTR Configs #74](https://buildkite.com/elastic/kibana-pull-request/builds/242003#01928a62-59b6-482e-acbc-ff7929386783)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/242003#01928a62-59b6-482e-acbc-ff7929386783) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/242003/jobs/01928a62-59b6-482e-acbc-ff7929386783/artifacts/01928a8b-bf2d-41cc-a12b-5d954d90d323) FTR Configs #<span></span>74 / discover/esql discover esql view switch modal should not show switch modal when switching to a data view while a saved search is open
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/242003#01928a62-597c-4989-8530-a2d8b36dab6b) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/242003/jobs/01928a62-597c-4989-8530-a2d8b36dab6b/artifacts/01928a86-84c2-4695-98d3-a1befde84bde) FTR Configs #<span></span>48 / Serverless Common UI - Platform Security Management navigation cards as Admin custom roles displays the Organization members management card, and will navigate to the cloud organization URL
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/242003#01928a62-598c-43e7-be48-6993307ceecf) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/242003/jobs/01928a62-598c-43e7-be48-6993307ceecf/artifacts/01928a87-6c9d-4261-904c-9f9ce58210f1) FTR Configs #<span></span>56 / Serverless Common UI - Platform Security Management navigation cards as Admin custom roles displays the Organization members management card, and will navigate to the cloud organization URL
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/242003#01928a87-7d2b-41e4-b3b2-fadcd886a8af) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/242003/jobs/01928a87-7d2b-41e4-b3b2-fadcd886a8af/artifacts/01928a9a-ac5a-4477-bc51-4534e84053f7) FTR Configs #<span></span>56 / Serverless Common UI - Platform Security Management navigation cards as Admin custom roles displays the Organization members management card, and will navigate to the cloud organization URL

### History
* :broken_heart: [Build #241972](https://buildkite.com/elastic/kibana-pull-request/builds/241972) failed 3dc49a470916ba0958a28410606a5ea59cfa8fa9
* :broken_heart: [Build #241949](https://buildkite.com/elastic/kibana-pull-request/builds/241949) failed e72cf087d6b1a2f4a0ad788b0d0201307a5ecdb8
* :green_heart: [Build #241517](https://buildkite.com/elastic/kibana-pull-request/builds/241517) succeeded 5cee6b25d96f5bc2f61c61bb40de56b536d7d45c
* :broken_heart: [Build #241426](https://buildkite.com/elastic/kibana-pull-request/builds/241426) failed 2bb076664ad69fd973837fb56ec3d3c7c882088a
* :broken_heart: [Build #241272](https://buildkite.com/elastic/kibana-pull-request/builds/241272) failed 1994ad6c7c2df9fae2fe8a53aa9539adaf8faf8e

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242003","number":242003,"commit":"3d77eaab53b3b3d7bf2a019c5756a9cde09653fb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241972","number":241972,"commit":"3dc49a470916ba0958a28410606a5ea59cfa8fa9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241949","number":241949,"commit":"e72cf087d6b1a2f4a0ad788b0d0201307a5ecdb8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241517","number":241517,"commit":"5cee6b25d96f5bc2f61c61bb40de56b536d7d45c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241426","number":241426,"commit":"2bb076664ad69fd973837fb56ec3d3c7c882088a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241272","number":241272,"commit":"1994ad6c7c2df9fae2fe8a53aa9539adaf8faf8e"}],"number":242003}
buildkite-pr-comment-->

eokoneyo reviewed (approved):
Shared UX changes LGTM

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/242501)
* Commit: f3e47b41f09fe51890c4addada0b6bb1e705df00

### Failed CI Steps
* [Jest Tests #7](https://buildkite.com/elastic/kibana-pull-request/builds/242501#01928fa8-af7b-47ee-a2f2-7bbb69b508ad)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/b93d3c224aeae33fa59482094c9927f0358c6ec8) | [after](https://github.com/elastic/kibana/commit/f3e47b41f09fe51890c4addada0b6bb1e705df00) | diff |
| --- | --- | --- | --- |
| `serverless` | 14.1KB | 14.1KB | +1.0B |



### History
* :broken_heart: [Build #242003](https://buildkite.com/elastic/kibana-pull-request/builds/242003) failed 3d77eaab53b3b3d7bf2a019c5756a9cde09653fb
* :broken_heart: [Build #241972](https://buildkite.com/elastic/kibana-pull-request/builds/241972) failed 3dc49a470916ba0958a28410606a5ea59cfa8fa9
* :broken_heart: [Build #241949](https://buildkite.com/elastic/kibana-pull-request/builds/241949) failed e72cf087d6b1a2f4a0ad788b0d0201307a5ecdb8
* :green_heart: [Build #241517](https://buildkite.com/elastic/kibana-pull-request/builds/241517) succeeded 5cee6b25d96f5bc2f61c61bb40de56b536d7d45c
* :broken_heart: [Build #241426](https://buildkite.com/elastic/kibana-pull-request/builds/241426) failed 2bb076664ad69fd973837fb56ec3d3c7c882088a

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242501","number":242501,"commit":"f3e47b41f09fe51890c4addada0b6bb1e705df00"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242003","number":242003,"commit":"3d77eaab53b3b3d7bf2a019c5756a9cde09653fb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241972","number":241972,"commit":"3dc49a470916ba0958a28410606a5ea59cfa8fa9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241949","number":241949,"commit":"e72cf087d6b1a2f4a0ad788b0d0201307a5ecdb8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241517","number":241517,"commit":"5cee6b25d96f5bc2f61c61bb40de56b536d7d45c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241426","number":241426,"commit":"2bb076664ad69fd973837fb56ec3d3c7c882088a"}],"number":242501}
buildkite-pr-comment-->

dmlemeshko commented on x-pack/test_serverless/shared/config.base.ts:
```diff
@@ -167,6 +167,15 @@ export default async () => {
         // configure security reponse header report-to settings to mimic MKI configuration
         `--csp.report_to=${JSON.stringify(['violations-endpoint'])}`,
         `--permissionsPolicy.report_to=${JSON.stringify(['violations-endpoint'])}`,
+        // normally below is injected by control plane
+        `--xpack.cloud.serverless.project_id=fakeprojectid`,
+        `--xpack.cloud.base_url=https://fake-cloud.elastic.co`,
+        `--xpack.cloud.projects_url=/projects/`,
+        `--xpack.cloud.profile_url=/user/settings/`,
+        `--xpack.cloud.billing_url=/billing/overview/`,
+        `--xpack.cloud.deployments_url=/deployments`,
+        `--xpack.cloud.organization_url=/account/`,
+        `--xpack.cloud.users_and_roles_url=/account/members/`,
```
Thanks a lot for addressing it!

dmlemeshko reviewed (approved):
appex-qa changes LGTM

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/242602)
* Commit: 3fbbf405da2f87b78a33b237194d17ed53988795

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/04efa04d6b8fc7ac6bc6996717453bd56200104a) | [after](https://github.com/elastic/kibana/commit/3fbbf405da2f87b78a33b237194d17ed53988795) | diff |
| --- | --- | --- | --- |
| `serverless` | 14.1KB | 14.1KB | +1.0B |



### History
* :yellow_heart: [Build #242501](https://buildkite.com/elastic/kibana-pull-request/builds/242501) was flaky f3e47b41f09fe51890c4addada0b6bb1e705df00
* :broken_heart: [Build #242003](https://buildkite.com/elastic/kibana-pull-request/builds/242003) failed 3d77eaab53b3b3d7bf2a019c5756a9cde09653fb
* :broken_heart: [Build #241972](https://buildkite.com/elastic/kibana-pull-request/builds/241972) failed 3dc49a470916ba0958a28410606a5ea59cfa8fa9
* :broken_heart: [Build #241949](https://buildkite.com/elastic/kibana-pull-request/builds/241949) failed e72cf087d6b1a2f4a0ad788b0d0201307a5ecdb8
* :green_heart: [Build #241517](https://buildkite.com/elastic/kibana-pull-request/builds/241517) succeeded 5cee6b25d96f5bc2f61c61bb40de56b536d7d45c
* :broken_heart: [Build #241426](https://buildkite.com/elastic/kibana-pull-request/builds/241426) failed 2bb076664ad69fd973837fb56ec3d3c7c882088a

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242602","number":242602,"commit":"3fbbf405da2f87b78a33b237194d17ed53988795"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242501","number":242501,"commit":"f3e47b41f09fe51890c4addada0b6bb1e705df00"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242003","number":242003,"commit":"3d77eaab53b3b3d7bf2a019c5756a9cde09653fb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241972","number":241972,"commit":"3dc49a470916ba0958a28410606a5ea59cfa8fa9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241949","number":241949,"commit":"e72cf087d6b1a2f4a0ad788b0d0201307a5ecdb8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241517","number":241517,"commit":"5cee6b25d96f5bc2f61c61bb40de56b536d7d45c"}],"number":242602}
buildkite-pr-comment-->

azasypkin commented:
__ACK:__ will review today

azasypkin commented on x-pack/test_serverless/api_integration/test_suites/common/management/spaces.ts:
```diff
@@ -34,76 +50,212 @@ export default function ({ getService }: FtrProviderContext) {
-            name: 'Custom',
-            disabledFeatures: [],
-          });
+    // The create and update test cases are unique to serverless becuase
```
__typo:__
```suggestion
    // The create and update test cases are unique to serverless because
```

azasypkin commented on x-pack/test_serverless/api_integration/test_suites/common/platform_security/authorization.ts:
```diff
@@ -8,8 +8,20 @@
 
+/*
+ * This file contains the common authorization tests that are applicable to
+ * all peroject types. There are two other authorization test files due to
+ * divergance of the custom roles feature:
+ *  - authorization_custom_roles.ts: custom roles are enabled in search and
+ *    security projects, so endpoints related to creating roles are enable.
```
__typo:__
```suggestion
 * all project types. There are two other authorization test files due to
 * divergence of the custom roles feature:
 *  - authorization_custom_roles.ts: custom roles are enabled in search and
 *    security projects, so endpoints related to creating roles are enabled.
```

azasypkin commented on x-pack/test_serverless/api_integration/test_suites/common/platform_security/authorization.ts:
```diff
@@ -8,8 +8,20 @@
 
+/*
+ * This file contains the common authorization tests that are applicable to
+ * all peroject types. There are two other authorization test files due to
```
__question:__ hmm, I didn't manage to find these files (`authorization_custom_roles.ts` and `authorization_oblt.ts`) 🙈 Where are they?

azasypkin commented on x-pack/test_serverless/functional/test_suites/observability/config.feature_flags.ts:
```diff
@@ -21,16 +21,11 @@ export default createTestConfig({
   // include settings from project controller
   // https://github.com/elastic/project-controller/blob/main/internal/project/observability/config/elasticsearch.yml
-  esServerArgs: ['xpack.ml.dfa.enabled=false', 'xpack.security.authc.native_roles.enabled=true'],
+  esServerArgs: ['xpack.ml.dfa.enabled=false'],
```
__question:__ When the O11y solution is ready to start working on custom roles functionality and would like to run tests related to roles regularly - locally, on CI, and in MKI - but aren’t ready to enable roles in production yet, what would be our recommendation? Will they need to remove "skip" flags and add the necessary overrides in GitOps to enable custom roles functionality in QA only to ensure we can run MKI tests?

jeramysoucy replied:
Good question. I guess we have a couple of options:
1. Retain the feature flag tests for OBLT to use (however, I don't think these get run on MKI)
2. Adjust skip flags and FTR config, enable in QA/Staging, keep disabled in prod

Maybe @pheyos has a better suggestion?

pheyos replied:
I think we only rarely want to enable feature flags broadly in a whole env for testing and custom roles for Observability does not really qualify for an exception there IMO. Here's what I think they should do once they're ready to test:

* get feature flag tests in - doesn't run on MKI but shows that things are functional
* create one or more test projects in MKI and override the setting for these projects in order to tests end-to-end
  * run the feature flag tests against this project

If that's all looking good:
* make sure no existing tests are breaking if custom roles are enabled
* step by step enabled the feature in QA, Staging and Production
* then move the feature flag tests to regular tests

jeramysoucy replied:
Thanks, Robert! @azasypkin Do you think we should we split out the roles tests and set the feature flag test back up for OBLT now? Not sure what their timeframe is for enabling custom roles.

azasypkin replied:
Thanks, @jeramysoucy. TL;DR: let’s keep what you have here.

Since there are no immediate signs that O11y is planning to introduce custom roles anytime soon (though I believe this might change quickly with GA), I don’t think we need to make the testing setup more complex than it needs to be. Once the O11y team is ready, we’ll propose what Robert outlined above and provide any necessary assistance if needed.

azasypkin reviewed (approved):
Great job! I left a few nits and one question regarding running custom roles tests for O11y once the team is ready to develop and test them.

jeramysoucy commented on config/serverless.yml:
```diff
@@ -225,3 +225,6 @@ monitoring.ui.enabled: false
 discover.enableUiSettingsValidations: true
+
+# Enable role management (custom roles) by default
+xpack.security.roleManagementEnabled: true
```
Good point. No, I don't think there is any benefit to having it here, other than maybe just visibility - but that is a stretch. I'll move it.

jeramysoucy commented:
Thanks @azasypkin! I addressed all feedback in [74639ce](https://github.com/elastic/kibana/pull/195584/commits/74639ceebdc7a909fbf2c068c4281d23f6841a72)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/244224)
* Commit: e663a712bc4afd9d08f83a24e1a0d2e4c9db5f11

### Failed CI Steps
* [FTR Configs #12](https://buildkite.com/elastic/kibana-pull-request/builds/244224#0192aeeb-5362-4283-9926-1260c5771bb6)
* [FTR Configs #12](https://buildkite.com/elastic/kibana-pull-request/builds/244224#0192af10-dad4-4728-b1cf-31c660fc6fda)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/244224#0192aeeb-5362-4283-9926-1260c5771bb6) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/244224/jobs/0192aeeb-5362-4283-9926-1260c5771bb6/artifacts/0192af10-c64d-4f1d-9609-09965993395e) FTR Configs #<span></span>12 / serverless observability UI - feature flags Role Management With custom role "after all" hook for "should access console with API key"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/244224#0192af10-dad4-4728-b1cf-31c660fc6fda) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/244224/jobs/0192af10-dad4-4728-b1cf-31c660fc6fda/artifacts/0192af1b-36b2-4df9-9e52-ebdd17ad151a) FTR Configs #<span></span>12 / serverless observability UI - feature flags Role Management With custom role "after all" hook for "should access console with API key"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/244224#0192aeeb-5362-4283-9926-1260c5771bb6) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/244224/jobs/0192aeeb-5362-4283-9926-1260c5771bb6/artifacts/0192af10-c64f-4db6-947c-b24ccb7dd708) FTR Configs #<span></span>12 / serverless observability UI - feature flags Role Management With custom role "before all" hook for "should have limited navigation menu"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/244224#0192af10-dad4-4728-b1cf-31c660fc6fda) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/244224/jobs/0192af10-dad4-4728-b1cf-31c660fc6fda/artifacts/0192af1b-36b1-4714-9708-c2b3902e9072) FTR Configs #<span></span>12 / serverless observability UI - feature flags Role Management With custom role "before all" hook for "should have limited navigation menu"

### History
* :green_heart: [Build #242602](https://buildkite.com/elastic/kibana-pull-request/builds/242602) succeeded 3fbbf405da2f87b78a33b237194d17ed53988795
* :yellow_heart: [Build #242501](https://buildkite.com/elastic/kibana-pull-request/builds/242501) was flaky f3e47b41f09fe51890c4addada0b6bb1e705df00
* :broken_heart: [Build #242003](https://buildkite.com/elastic/kibana-pull-request/builds/242003) failed 3d77eaab53b3b3d7bf2a019c5756a9cde09653fb
* :broken_heart: [Build #241972](https://buildkite.com/elastic/kibana-pull-request/builds/241972) failed 3dc49a470916ba0958a28410606a5ea59cfa8fa9
* :broken_heart: [Build #241949](https://buildkite.com/elastic/kibana-pull-request/builds/241949) failed e72cf087d6b1a2f4a0ad788b0d0201307a5ecdb8
* :green_heart: [Build #241517](https://buildkite.com/elastic/kibana-pull-request/builds/241517) succeeded 5cee6b25d96f5bc2f61c61bb40de56b536d7d45c

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/244224","number":244224,"commit":"e663a712bc4afd9d08f83a24e1a0d2e4c9db5f11"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242602","number":242602,"commit":"3fbbf405da2f87b78a33b237194d17ed53988795"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242501","number":242501,"commit":"f3e47b41f09fe51890c4addada0b6bb1e705df00"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242003","number":242003,"commit":"3d77eaab53b3b3d7bf2a019c5756a9cde09653fb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241972","number":241972,"commit":"3dc49a470916ba0958a28410606a5ea59cfa8fa9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241949","number":241949,"commit":"e72cf087d6b1a2f4a0ad788b0d0201307a5ecdb8"}],"number":244224}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/244264)
* Commit: e9462c3ae42371444a87e0fc74d1da69c4d6c101

### Failed CI Steps
* [Jest Tests #18](https://buildkite.com/elastic/kibana-pull-request/builds/244264#0192af42-5c73-47ba-bf68-794f5f682de1)
* [Jest Tests #18](https://buildkite.com/elastic/kibana-pull-request/builds/244264#0192af72-7721-4080-83dd-334f373d4e29)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/244264#0192af42-5c73-47ba-bf68-794f5f682de1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/244264/jobs/0192af42-5c73-47ba-bf68-794f5f682de1/artifacts/0192af72-25dd-4790-a837-288cb9d0e476) Jest Tests #<span></span>18 / config schema generates proper defaults
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/244264#0192af72-7721-4080-83dd-334f373d4e29) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/244264/jobs/0192af72-7721-4080-83dd-334f373d4e29/artifacts/0192af80-28ed-4a6a-82cb-86b1a28512d1) Jest Tests #<span></span>18 / config schema generates proper defaults

### History
* :broken_heart: [Build #244224](https://buildkite.com/elastic/kibana-pull-request/builds/244224) failed e663a712bc4afd9d08f83a24e1a0d2e4c9db5f11
* :green_heart: [Build #242602](https://buildkite.com/elastic/kibana-pull-request/builds/242602) succeeded 3fbbf405da2f87b78a33b237194d17ed53988795
* :yellow_heart: [Build #242501](https://buildkite.com/elastic/kibana-pull-request/builds/242501) was flaky f3e47b41f09fe51890c4addada0b6bb1e705df00
* :broken_heart: [Build #242003](https://buildkite.com/elastic/kibana-pull-request/builds/242003) failed 3d77eaab53b3b3d7bf2a019c5756a9cde09653fb
* :broken_heart: [Build #241972](https://buildkite.com/elastic/kibana-pull-request/builds/241972) failed 3dc49a470916ba0958a28410606a5ea59cfa8fa9

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/244264","number":244264,"commit":"e9462c3ae42371444a87e0fc74d1da69c4d6c101"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/244224","number":244224,"commit":"e663a712bc4afd9d08f83a24e1a0d2e4c9db5f11"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242602","number":242602,"commit":"3fbbf405da2f87b78a33b237194d17ed53988795"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242501","number":242501,"commit":"f3e47b41f09fe51890c4addada0b6bb1e705df00"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242003","number":242003,"commit":"3d77eaab53b3b3d7bf2a019c5756a9cde09653fb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/241972","number":241972,"commit":"3dc49a470916ba0958a28410606a5ea59cfa8fa9"}],"number":244264}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/244324)
* Commit: 504802beb874c6e661804adbcaebe7e2d1e2d54e

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/68b3267ca2f76d8de4fedef622608d8685e7eceb) | [after](https://github.com/elastic/kibana/commit/504802beb874c6e661804adbcaebe7e2d1e2d54e) | diff |
| --- | --- | --- | --- |
| `serverless` | 14.1KB | 14.1KB | +1.0B |



### History
* :broken_heart: [Build #244264](https://buildkite.com/elastic/kibana-pull-request/builds/244264) failed e9462c3ae42371444a87e0fc74d1da69c4d6c101
* :broken_heart: [Build #244224](https://buildkite.com/elastic/kibana-pull-request/builds/244224) failed e663a712bc4afd9d08f83a24e1a0d2e4c9db5f11
* :green_heart: [Build #242602](https://buildkite.com/elastic/kibana-pull-request/builds/242602) succeeded 3fbbf405da2f87b78a33b237194d17ed53988795
* :yellow_heart: [Build #242501](https://buildkite.com/elastic/kibana-pull-request/builds/242501) was flaky f3e47b41f09fe51890c4addada0b6bb1e705df00
* :broken_heart: [Build #242003](https://buildkite.com/elastic/kibana-pull-request/builds/242003) failed 3d77eaab53b3b3d7bf2a019c5756a9cde09653fb
* :broken_heart: [Build #241972](https://buildkite.com/elastic/kibana-pull-request/builds/241972) failed 3dc49a470916ba0958a28410606a5ea59cfa8fa9

cc @jeramysoucy

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/244324","number":244324,"commit":"504802beb874c6e661804adbcaebe7e2d1e2d54e"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/244264","number":244264,"commit":"e9462c3ae42371444a87e0fc74d1da69c4d6c101"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/244224","number":244224,"commit":"e663a712bc4afd9d08f83a24e1a0d2e4c9db5f11"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242602","number":242602,"commit":"3fbbf405da2f87b78a33b237194d17ed53988795"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242501","number":242501,"commit":"f3e47b41f09fe51890c4addada0b6bb1e705df00"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/242003","number":242003,"commit":"3d77eaab53b3b3d7bf2a019c5756a9cde09653fb"}],"number":244324}
buildkite-pr-comment-->
